modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGeneratorBaseController < ApplicationController
+      service_tag 'representation-management'
+      skip_before_action :authenticate
+      before_action :feature_enabled
+
+      private
+
+      def params_permitted
+        [
+          :record_consent,
+          :consent_address_change,
+          { consent_limits: [],
+            conditions_of_appointment: [],
+            claimant: claimant_params_permitted,
+            veteran: veteran_params_permitted }
+        ]
+      end
+
+      def claimant_params_permitted
+        [
+          :date_of_birth,
+          :relationship,
+          :phone,
+          :email,
+          { name: %i[
+              first
+              middle
+              last
+            ],
+            address: %i[
+              address_line1
+              address_line2
+              city
+              state_code
+              country
+              zip_code
+              zip_code_suffix
+            ] }
```

Comment:
Name and address can be extracted to 

```
def name_params_permitted
  %i[first middle last]
end

def address_params_permitted
  %i[
    address_line1
    address_line2
    city
    state_code
    country
    zip_code
    zip_code_suffix
  ]
end
```

Then you can use `name: name_params_permitted` and `address: address_params_permitted` in: 
- `claimant_params_permitted`
- `veteran_params_permitted`
- `representative_params_permitted`

Reply:
Sounds good!